service/controller/clusterapi/v30/resource/tccp/testdata/case-0-basic-test.golden

```diff
@@ -528,6 +528,37 @@ Resources:
       Tags:
         - Key: Name
           Value: 8y5ck-etcd-elb
+  APIInternalELBSecurityGroup:
+    Type: AWS::EC2::SecurityGroup
+    Properties:
+      GroupDescription: 8y5ck-internal-api
+      VpcId: !Ref VPC
+      SecurityGroupIngress:
+      
```

Comment:
Been thinking about it but the rendering here doesn't make my eyes particularly happy.
It's fine for now as it is consistent at least, but I might try and play around with it in a follow up.

Reply:
I’m not much into details on this. Can I improve that somehow?

Reply:
I played around briefly but can't make it better - so all good.